### PR TITLE
fix #11

### LINKED ISSUE #11: cd to subdirectory of ~ does not work

### PR DESCRIPTION
**Pull request by sadanslargehole**
expand home path in cd

### CODE REVIEW

**Comment by keli5** [comment]
Will review later if this fixes #11 . #8 is out of scope for this PR.

**Comment by sadanslargehole** [comment]
Will separate when i can

**Review by keli5 — COMMENTED** [review]
This looks fine? Something like "cd ~/dir/.." doesn't seem to work - though that's very niche, shell scripting only maybe, and honestly might be my environment... will look more later.

File: main.cpp [hunk 1]
```diff
@@ -75,6 +75,8 @@ int main() {
                 fs::current_path(dest);
                 setenv("PWD", fs::current_path().c_str(), 1);
                 continue;
+            }else if (params.at(0)=='~'){
+                replaceSubstring(params, "~", homePath());
```
Comment (keli5) [inline main.cpp:79]: As far as I can tell, this still doesn't work?.. On my machine, I have a `projects` folder in my home folder. I still get `cd: No such file or directory` if i run `cd ~/projects`.

**Review by keli5 — CHANGES_REQUESTED** [review]
